### ISSUE TITLE
Is there a key that will clear all the data?

### ISSUE THREAD

**Issue by MiloMindbender** [farrellf/TelemetryViewer#47]
Is there a function to clear all the captured data and start fresh?  Sometimes when testing stuff I just want to throw all the data out before starting the next run so the data doesn't get mixed together.  So far the only way I see to do this is re-import the settings file or restart the program.

**Comment by farrellf** [comment]
You're right, that's the only way to do it right now. I'll think about adding this.
